File: client/Client/Network/Messages/Game/LifePointsUpdateMessage.cs
namespace LoadRunnerClient.Messages
{
    public class LifePointsUpdateMessage
    {
        public const string TYPE = "LifePointsUpdateMessage";

        public LifePointsUpdateMessage()
        {
        }

        /// <summary>
        /// LifePointsUpdateMessage Constructor
        /// </summary>
        /// <param name="uuid"></param>
        /// <param name="lifePoints"></param>
        public LifePointsUpdateMessage(string uuid, int lifePoints)
        {
            this._uuid = uuid;
            this._lifePoints = lifePoints;
        }

        private string _uuid;

        public string uuid
        {
            get => _uuid;
            set => _uuid = value;
        }

        private int _lifePoints;

        public int lifePoints
        {
            get => _lifePoints;
            set => _lifePoints = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/LifePointsUpdateMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;

public class LifePointsUpdateMessage {

	public static final String TYPE = "LifePointsUpdateMessage";

	private String uuid;
	private int lifePoints;

	public LifePointsUpdateMessage() {
	}

	public LifePointsUpdateMessage(String uuid, int lifePoints) {
		this.uuid = uuid;
		this.lifePoints = lifePoints;
	}

	public String getUuid() {
		return uuid;
	}

	public void setUuid(String uuid) {
		this.uuid = uuid;
	}

	public int getLifePoints() {
		return lifePoints;
	}

	public void setLifePoints(int lifePoints) {
		this.lifePoints = lifePoints;
	}
}
